x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -63,6 +63,26 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         expect(await newComment.getVisibleText()).equal('Test comment from automation');
       });
 
+      it('adds a category to a case', async () => {
+        const category = uuidv4();
+        await testSubjects.click('category-edit-button');
+        await comboBox.setCustom('comboBoxInput', category);
+        await testSubjects.click('edit-category-submit');
+
+        // validate category was added
+        await testSubjects.existOrFail('category-' + category);
+
+        // validate user action
+        await find.byCssSelector('[data-test-subj*="category-update-action"]');
+      });
+
+      it('deletes a category from a case', async () => {
+        await testSubjects.click('category-remove-button');
```

Comment:
We should validate in the begging of the test that the category exists like `await testSubjects.existOrFail('category-' + category);`